puara_gestures.h

```diff
@@ -73,14 +82,39 @@ class PuaraGestures {
     public:
         float leakyIntegrator (float reading, float old_value, float leak, int frequency, unsigned long& timer);
         
+        // Inertial measurement updates (accelerometer, gyroscope, magnetometer)
+        void updateAccel(float accelX, float accelY, float accelZ); // in m/sec^2
+        void updateGyro(float gyroX, float gyroY, float gyroZ); // in rad/sec
+        void updateMag(float magX, float magY, float magZ); // in uTesla
+        void updateInertialGestures(); // Updates shake, jab and orientation
+
+        // General inertial sensor signals
+        float getAccelX(); // in m/s^2, converted from g's
+        float getAccelY();
+        float getAccelZ();
+        float getGyroX(); // in radians per second, converted from DPS
+        float getGyroY();
+        float getGyroZ();
+        float getMagX(); // in uTesla, converted from Gauss
+        float getMagY();
+        float getMagZ();
+        float getYaw();
+        float getPitch();
+        float getRoll();
+
         // Shake and Jab
-        void updateJabShake (float gyroX, float gyroY, float gyroZ); // in radians per second
         float getShakeX();
         float getShakeY();
         float getShakeZ();
         float getJabX();
         float getJabY();
         float getJabZ();
+
+        // Orientation
+        double getOrientationW();
+        double getOrientationX();
+        double getOrientationY();
+        double getOrientationZ();
```

Comment:
Everything looks good.

My only comment/suggestion is regarding the function's name. Maybe it would make more sense to have a function such as `getQuaternionW`, `getQuaternionX`, etc. Orientation seems a bit ambiguous to me. What do you think?

Then it would be something like:

```suggestion
        double getQuaternionW();
        double getQuaternionX();
        double getQuaternionY();
        double getQuaternionZ();
```

Reply:
On first thought I'd agree with you, but on second thought we're exposing a signal called "orientation", so getOrientation would match up with that API. Maybe we could add the unit of orientation to be quaternion to clear things up? If you agree, I'll add function comments to clear things up for developers too.

Reply:
That's reasonable. I assume we'll keep the OSC namespaces so no change for the end user.